bitnami/opencart/README.md

```diff
@@ -48,74 +48,147 @@ The command removes all the Kubernetes components associated with the chart and
+| `readinessProbe`                     | Readiness probe configuration                                                                                         | `Check values.yaml file`                       |
+| `replicaCount`                       | Number of OpenCart Pods to run                                                                                        | `1`                                            |
+| `resources`                          | CPU/Memory resource requests/limits                                                                                   | Memory: `512Mi`, CPU: `300m`                   |
+| `sidecars`                           | Attach additional containers to the pod (evaluated as a template)                                                     | `nil`                                          |
+| `smtpHost`                           | SMTP host                                                                                                             | `nil`                                          |
+| `smtpPort`                           | SMTP port                                                                                                             | `nil` (but opencart internal default is 25)    |
+| `smtpProtocol`                       | SMTP Protocol (options: ssl,tls, nil)                                                                                 | `nil`                                          |
+| `smtpUser`                           | SMTP user                                                                                                             | `nil`                                          |
+| `smtpPassword`                       | SMTP password                                                                                                         | `nil`                                          |
+| `tolerations`                        | Tolerations for pod assignment                                                                                        | `[]` (The value is evaluated as a template)    |
+| `updateStrategy`                     | Deployment update strategy                                                                                            | `nil`                                          |
+
+### Traffic Exposure Parameters
+
+| Parameter                        | Description                           | Default            |
+|----------------------------------|---------------------------------------|--------------------|
+| `service.type`                   | Kubernetes Service type               | `LoadBalancer`     |
+| `service.port`                   | Service HTTP port                     | `8080`             |
+| `service.httpsPort`              | Service HTTPS port                    | `8443`             |
```

Comment:
These don't match the values.yaml defaults

Reply:
good catch, fixing